composer.json

```diff
@@ -16,9 +16,27 @@
         }
     },
     "require": {
-        "php": "^7.0"
+        "php": "^7.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.5"
+        "phpunit/phpunit": "^8.1",
```

Comment:
phpunit 8 requires php 7.2 .. 🤨

Reply:
I see... So let's bump minimal php version :upside_down_face: ? Or we will be more conservative and keep compatibility? 

IMO who has an older PHP can still use any previous version of this library, so I'd say its ok to require 7.2+.